### ISSUE TITLE
Accessibility Improvements

### ISSUE THREAD

**Issue by seanrucker** [RoverPlatform/rover-ios#524]
1. Text elements are read as disabled text fields (dimmed means disabled) but should be read as plain text labels.
2. Rover blocks with tappable actions (buttons or tappable images) need to have a trait set that indicates they are tappable.

See video in Dropbox > Engineering > issues > rover-ios > 524
